tests/tests_reload_on_reset.yml

```diff
@@ -21,9 +21,11 @@
           set_fact:
             __firewall_is_ostree: "{{ __ostree_booted_stat.stat.exists }}"
 
-    - name: Install podman
+    - name: Install test dependencies
       package:
-        name: podman
+        name:
+          - podman
+          - /usr/bin/ping
```

Comment:
This introduces a dependency at ostree build time - either some package needs to be added to https://github.com/linux-system-roles/firewall/blob/main/.ostree/packages-testing.txt or possibly one of the platform specific files.  I don't know if image builder can use a file, or if it requires the name of a package.

Reply:
Ack! Done.